lang/LangSource/SC_LanguageConfig.cpp

```diff
@@ -93,47 +103,53 @@ bool SC_LanguageConfig::removeIncludedDirectory(const Path& path) { return remov
 
 bool SC_LanguageConfig::removeExcludedDirectory(const Path& path) { return removePath(mExcludedDirectories, path); }
 
+static void processPathList(const char* nodeName, YAML::Node& doc,
+                            const std::function<void(const boost::filesystem::path&)>& func) {
+    const YAML::Node& items = doc[nodeName];
+    if (items && items.IsSequence()) {
+        for (auto const& item : items) {
+            const std::string& path = item.as<std::string>("");
+            if (!path.empty()) {
+                const boost::filesystem::path& native_path = SC_Codecvt::utf8_str_to_path(path);
+                func(native_path);
+            }
+        }
+    }
+}
+
+static void processBool(const char* nodeName, YAML::Node& doc, const std::function<void(bool)>& successFunc,
+                        const std::function<void(void)>& failFunc) {
```

Comment:
write this as `std::function<void()>`, the inner void is redundant

edit - did not intend for that to sound so philosophical haha